src/query.md

```diff
@@ -10,7 +10,7 @@ the [def-id] of some item, will compute the type of that item and return
 it to you.
 
 [def-id]: appendix/glossary.md#def-id
-[hl]: high-level-overview.html
+[hl]: overview.html
```

Comment:
Given from git blame, I think it should be https://github.com/rust-lang/rustc-dev-guide/blob/master/src/compiler-src.md.